Importer/Interpreter/EntityDataInterpreter.php

```diff
@@ -120,22 +122,17 @@ public function interpret(array $data, $associative = true)
                 $entities[$index] = $entity;
                 $this->internalLookupCache[] = $entity;
             } catch (InterpreterException $exception) {
-                $this->handleInterpreterError($exception, $index, $row);
+                $this->eventDispatcher->dispatch(ImportEvents::DATA_ERROR, new DataErrorEvent($exception, $index, $this->log));
+                if ($this->log->getErrorCount() >= $this->log->getMaxErrorCount()) {
```

Comment:
It would be better to know whether we should continue or halt. Otherwise we will have to call a `getItIsAfter6pm()` method as well if we want to stop processing after 6pm (or any rule we could think of).